index.html

```diff
@@ -101,12 +101,12 @@ <h3 class="widget-title">Become a member!</h3>
         </div>
         <!--End mc_embed_signup-->
 
-        <aside id="reallysimpletwitterwidget-2" class="widget widget_reallysimpletwitterwidget"><h3 class="widget-title">Stay Connected!</h3>
-            <div class="socialbuttons col-center">  
-                <!-- Facebook Button -->
-                <div class="fb-follow" data-href="https://www.facebook.com/pages/TRUSU-Computer-Science-Club/343428029131317?fref=ts" data-colorscheme="light" data-layout="button" data-show-faces="true"></div>
-                <!-- Twitter Button -->
-                <a class="twitter-follow-button" href="https://twitter.com/trucsclub" data-show-count="false" data-lang="en">Follow @trucsclub</a>  
+        <aside id="reallysimpletwitterwidget-2" class="widget widget_reallysimpletwitterwidget">
+            <h3 class="widget-title">Stay Connected!</h3>
```

Comment:
Agreed. I'd love to do some more cleanup as well. Supposedly one of our asides doesn't have an ending. More cleanup could be done in a separate PR though. I'll fix these.